test/dx_e2e.js

```diff
@@ -1,56 +1,115 @@
+
+    // do the necessary fundings
+    const oneEth = new BN("1000000000000000000")
+    const oneGwei = new BN("10000000000000000")
+    await depositToken.deposit({ value: oneEth })
+    await depositToken.approve(instance1.address, oneEth)
     await depositToken.approve(dx.address, oneEth)
 
     await secondaryTokenMock.givenAnyReturnBool(true)
 
-    await instance.deposit(10)
-    assert.equal(await instance.numberOfParticipations.call(accounts[0]), 1)
+    await instance1.deposit(10)
+    assert.equal(await instance1.numberOfParticipations.call(accounts[0]), 1)
 
     //frist tokenPair needs to be funded first...
-    await dx.deposit(depositToken.address, oneGwei.toString())
-    await dx.deposit(secondaryTokenMock.address, oneGwei.toString())
+    await dx.deposit(depositToken.address, oneGwei.mul(new BN("5")).toString())
+    await dx.deposit(secondaryTokenMock.address, oneGwei.mul(new BN("5")).toString())
     await dx.addTokenPair(
       depositToken.address,
       secondaryTokenMock.address,
       oneGwei.toString(),
       oneGwei.toString(),
       1,
-      10
+      1
     )
-    //  await waitUntilPriceIsXPercentOfPreviousPrice(depositToken, secondaryTokenMock, 1);
+    // ensure that sells go into auction 2, as auction 1 will have been started
+    await increaseTimeBy(60 * 60 * 6)
+    console.log(await timestamp())
+    await coordinator.participateInAuction()
+
+    assert.equal(await dx.sellerBalances.call(depositToken.address, secondaryTokenMock.address, 2, instance1.address), 10)
+    assert.equal(await dx.sellerBalances.call(secondaryTokenMock.address, depositToken.address, 2, instance1.address), 0)
+
+    //closes auction 1
+    await waitUntilPriceIsXPercentOfPreviousPrice(depositToken, secondaryTokenMock, 1);
+    await dx.postBuyOrder(depositToken.address, secondaryTokenMock.address, 1, oneGwei.toString())
+    await dx.postBuyOrder(secondaryTokenMock.address, depositToken.address, 1, oneGwei.toString())
+
+    assert.equal(await dx.getAuctionIndex(depositToken.address, secondaryTokenMock.address),2)
+
+    //claim funds  and post into next auction auction with more
+    await dx.claimSellerFunds(depositToken.address, secondaryTokenMock.address, accounts[0], 1)
+    await dx.claimSellerFunds(secondaryTokenMock.address, depositToken.address, accounts[0], 1)
+    await dx.claimBuyerFunds(depositToken.address, secondaryTokenMock.address, accounts[0], 1)
+    await dx.claimBuyerFunds(secondaryTokenMock.address, depositToken.address, accounts[0], 1)
+    await dx.postSellOrder(depositToken.address, secondaryTokenMock.address, 0, oneGwei.toString())
+    await dx.postSellOrder(secondaryTokenMock.address, depositToken.address, 0, oneGwei.toString())
+
+    //close the auction and pool paritcpation in next one
+    await waitUntilPriceIsXPercentOfPreviousPrice(depositToken, secondaryTokenMock, 1);
+    await dx.postBuyOrder(depositToken.address, secondaryTokenMock.address, 2, oneGwei.toString())
```

Comment:
I don't understand why there seems to be an extra auction, in which we don't participate. Is this just so we test that this works or is there more to it? Could you document it more? I'm confused, because it seems to happen below as well.

Reply:
I all the time have to run the auction WETH-GNO and GNO-WETH. I just can't have a single one of them.